### ISSUE TITLE
Won't compile with Dagger 2.17

### ISSUE THREAD

**Issue by jasonhartley** [PhilippeBoisney/GithubArchitectureComponents#3]
Dagger 2.17 fixes a bug with @ Bind that can break builds.
https://github.com/google/dagger/wiki/Dagger-2.17-@Binds-bugs

**Comment by tatsuya-fujisaki** [comment]
Updating `AppComponent.java` fixed the error with Dagger 2.17.

Before:
```java
@Component(modules = {ActivityModule.class, FragmentModule.class, AppModule.class})
public interface AppComponent {
    ...
}
```

After:
```java
@Component(modules = {AndroidSupportInjectionModule.class, ActivityModule.class, AppModule.class})
public interface AppComponent {
    ...
}
```

**Comment by PhilippeBoisney** [comment]
Fixed
